javascript-conditionals/javascript-conditionals-quiz-notes.md

```diff
@@ -0,0 +1,49 @@
+# javascript-conditionals-quiz-notes
+
+## Quiz Questions
+
+After completing this exercise, you should be able to discuss or answer the following questions:
+
+- Give 6 examples of comparison operators.
+Greater than >, Less than <, Equal to ==, Stricly equal to ===, Not equal to !=, Less than or equal to <=
+- What data type do comparison expressions evaluate to?
+Values or Variables
```

Comment:
I would adjust this answer. Think of primitive data types. (String, number, etc)

Reply:
Are you saying I should adjust "Give 6 examples of comparison operators" or "What data type do comparison expressions evaluate to?"